docs/source/python_api/mlflow.metrics.rst

```diff
@@ -1,7 +1,58 @@
 mlflow.metrics
 ==============
 
+The ``mlflow.metrics`` module helps you quantitatively and qualitatively measure your models. These :py:class:`EvaluationMetric <mlflow.metrics.EvaluationMetric>` are used by the :py:func:`mlflow.evaluate()` API, either computed automatically depending on the ``model_type`` or specified via the ``custom_metrics`` parameter. Evaluation results are stored as :py:class:`MetricValue <mlflow.metrics.MetricValue>` and are logged to the MLflow run.
```

Comment:
Can we add code example(s) demonstrating how to use `mlflow.evaluate()` with the intelligent metrics?